packages/framework/fluid-framework/src/index.ts

```diff
@@ -54,13 +54,21 @@ export type {
 	ReplaceIEventThisPlaceHolder,
 	FluidObject, // Linked in doc comment
 	FluidObjectProviderKeys, // Used by FluidObject
+	// eslint-disable-next-line import/export
+	Listeners,
+	// eslint-disable-next-line import/export
+	IsListener,
+	// eslint-disable-next-line import/export
+	Listenable,
+	// eslint-disable-next-line import/export
+	Off,
 } from "@fluidframework/core-interfaces";
 
 export type { isFluidHandle } from "@fluidframework/runtime-utils";
 
 // Let the tree package manage its own API surface.
 // Note: this only surfaces the `@public, @beta and @alpha` API items from the tree package.
-// eslint-disable-next-line no-restricted-syntax, import/no-internal-modules
+// eslint-disable-next-line no-restricted-syntax, import/no-internal-modules, import/export
 export * from "@fluidframework/tree/alpha";
```

Comment:
Yes. It errors out saying multiple exports are with the same name (Listenable, Off, etc exports from tree as well as core-interfaces). One option is adding aliases. 

`Listenable as Listenable_base`

Reply:
Ugh. This is a pain. I hadn't considered how the `fluid-framework` re-exports would mess with things here. The easiest solution might just be to add a `@remarks` block or something to the moved APIs noting that they will be removed from `@fluidframework/tree`'s exports in 3.0, and that users should either import from `fluid-framework` or from `@fluidframework/core-utils`.

That comment would appear in the docs regardless of where the user is importing the APIs from, but it clarifies the expectation reasonably IMO.

Reply:
I tried importing these types in an example, and there were no tsc errors or strikethrough warnings from VS Code. This seems to be because, while the types are marked as deprecated in tree, they are still exported via fluid-framework from core-interfaces.
 
If the exports from core-interfaces (referenced in fluid-framework/index.ts) are removed, the usual strikethrough appears, reflecting their deprecation in tree.

Reply:
Thoughts on possible solution to resolve the double export that still is suspect (full implications unknown):

** If tree doesn't actually export any `@internal` APIs other than `configuredSharedTree` ?? **
In tree move all index.ts exports except these 4 events ones, `SharedTree` (`@legacy~alpha`), and any(?) internals and to index_common_api.ts. Then index.ts will be:
```typescript
export * from "./index_common_apis.js";

export {
	SharedTree,
	configuredSharedTree,
} from "./treeFactory.js";

export type {
	/**
	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
	 */
	Listeners,
	/**
	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
	 */
	IsListener,
	/**
	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
	 */
	Listenable,
	/**
	 * @deprecated Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or `@fluidframework/core-interfaces` instead.
	 */
	Off,
} from "@fluidframework/core-interfaces";
```
To tree package.json "exports" add:
```json
		"./internal/alpha-apis": {
			"import": {
				"types": "./lib/index_common_api.d.ts",
				"default": "./lib/index_common_api.js"
			},
			"require": {
				"types": "./dist/index_common_api.d.ts",
				"default": "./dist/index_common_api.js"
			}
		},
```
And finally in fluid-framework index.ts the tree/alpha export will become:
```typescript
export * from "@fluidframework/tree/internal/alpha-apis";
```
I think it would be possible to use a non-"/internal" path for tree export. That would add some validation with some extra script entries, but doesn't feel needed.

Reply:
I see api report failures locally with the above change after rebuilding the repo

```ts
fluid-framework: error during command 'api-extractor run --config api-extractor/api-extractor-lint-legacy.cjs.json' (exit code 1)
fluid-framework: api-extractor 7.47.8  - https://api-extractor.com/
fluid-framework: 
fluid-framework: Analysis will use the bundled TypeScript version 5.4.2
fluid-framework: 
fluid-framework: API Extractor completed with errors
fluid-framework: 
fluid-framework: Error: /workspaces/FluidFramework/packages/dds/tree/src/simple-tree/api/tree.ts:513:2 - (ae-forgotten-export) The symbol "Listenable" needs to be exported by the entry point legacy.d.ts
[20/24] x fluid-framework: api-extractor run --config api-extractor/api-extractor-lint-legacy.cjs.json - 5.062s
fluid-framework: error during command 'api-extractor run --local --config api-extractor/api-extractor.legacy.json' (exit code 1)
fluid-framework: api-extractor 7.47.8  - https://api-extractor.com/
fluid-framework: 
fluid-framework: Analysis will use the bundled TypeScript version 5.4.2
fluid-framework: 
fluid-framework: API Extractor completed with errors
fluid-framework: 
fluid-framework: Error: /workspaces/FluidFramework/packages/dds/tree/src/simple-tree/api/tree.ts:513:2 - (ae-forgotten-export) The symbol "Listenable" needs to be exported by the entry point legacy.d.ts
[21/24] x fluid-framework: api-extractor run --local --config api-extractor/api-extractor.legacy.json - 5.064s
fluid-framework: error during command 'api-extractor run --config api-extractor/api-extractor-lint-alpha.esm.json' (exit code 1)
fluid-framework: api-extractor 7.47.8  - https://api-extractor.com/
fluid-framework: 
fluid-framework: Analysis will use the bundled TypeScript version 5.4.2
fluid-framework: 
fluid-framework: API Extractor completed with errors
fluid-framework: 
fluid-framework: Error: /workspaces/FluidFramework/packages/dds/tree/src/simple-tree/api/tree.ts:380:2 - (ae-forgotten-export) The symbol "Listenable" needs to be exported by the entry point alpha.d.ts
[22/24] x fluid-framework: api-extractor run --config api-extractor/api-extractor-lint-alpha.esm.json - 5.127s
fluid-framework: error during command 'api-extractor run --local --config api-extractor/api-extractor.current.json' (exit code 1)
fluid-framework: api-extractor 7.47.8  - https://api-extractor.com/
fluid-framework: 
fluid-framework: Analysis will use the bundled TypeScript version 5.4.2
fluid-framework: 
fluid-framework: API Extractor completed with errors
fluid-framework: 
fluid-framework: Error: /workspaces/FluidFramework/packages/dds/tree/src/simple-tree/api/tree.ts:380:2 - (ae-forgotten-export) The symbol "Listenable" needs to be exported by the entry point alpha.d.ts
[23/24] x fluid-framework: api-extractor run --local --config api-extractor/api-extractor.current.json - 5.260s
```